src/views/CashlinkCreate.vue

```diff
@@ -0,0 +1,567 @@
+                shopOrigin: this.rpcState.origin,
+                appName: this.request.appName,
+
+                keyId: senderAccount.keyId,
+                keyPath: senderAccount.findSignerForAddress(this.accountOrContractInfo!.address)!.path,
+                keyLabel: senderAccount.label,
+
+                sender: this.accountOrContractInfo!.address.serialize(),
+                senderType: (this.accountOrContractInfo as ContractInfo).type
+                    ? (this.accountOrContractInfo as ContractInfo).type
+                    : Nimiq.Account.Type.BASIC,
+                senderLabel: this.accountOrContractInfo!.label,
+                recipientLabel: 'New Cashlink',
+                recipientType: Nimiq.Account.Type.BASIC,
+                fee: this.fee,
+                validityStartHeight,
+            });
+            staticStore.keyguardRequest = request;
+            staticStore.cashlink = cashlink!;
+            const client = this.$rpc.createKeyguardClient();
+            client.signTransaction(request);
+        });
+    }
+
+    private closeOptions() {
+        this.optionsOpened = false;
+        Vue.nextTick(() => (this.$refs.amountWithFee as AmountWithFee).focus());
+    }
+
+    private closeDetails() {
+        this.details = Details.NONE;
+        Vue.nextTick(() => (this.$refs.amountWithFee as AmountWithFee).focus());
+    }
+
+    private login() {
+        staticStore.originalRouteName = RequestType.CASHLINK;
+        this.$rpc.routerPush(RequestType.ONBOARD);
+    }
+
+    private close() {
+        this.$rpc.reject(new Error(ERROR_CANCELED));
+    }
+
+    private data() {
+            return {
+                OPTIONS: [{
+                    color: 'nq-light-blue-bg',
+                    value: 0,
+                    text: 'free',
+                    index: 0,
+                }, {
+                    color: 'nq-green-bg',
+                    value: 1,
+                    text: 'standard',
+                    index: 1,
+                }, {
+                    color: 'nq-gold-bg',
+                    value: 2,
+                    text: 'express',
+                    index: 2,
+                }],
+                Details,
+            };
+        }
+}
+</script>
+
+<style scoped>
+    .create-cashlink {
+        position: relative;
+    }
+
+    .create-cashlink .page-header,
+    .create-cashlink .page-footer {
+        transition: opacity .3s;
+        opacity: 1;
+    }
+
+    .page-footer .nq-button {
+        margin-top: 0;
+    }
+
+    .create-cashlink .page-body {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: space-around;
+        transition: filter .4s, opacity .3s;
+        opacity: 1;
+        padding-bottom: 2rem;
+        padding-top: 0;
+        -webkit-filter: blur(0px);
+        -moz-filter: blur(0px);
+        -o-filter: blur(0px);
+        -ms-filter: blur(0px);
+        filter: blur(0px);
```

Comment:
Manually prefixing is not necessary, the Vue build system does this automatically already with the `autoprefixer` plugin.